wolfssl/wolfcrypt/hmac.h

```diff
@@ -43,6 +43,10 @@
     WOLFSSL_LOCAL int wolfCrypt_FIPS_HMAC_sanity(void);
 #endif
 
+#if FIPS_VERSION3_GE(6,0,0)
+    #define FIPS_ALLOW_SHORT 1
```

Comment:
Should this new FIPS macro include HMAC in the name? Would be nice if `wc_HmacSetKey_ex` was unsigned and actually a flag / bit mask. Currently its 0 or non-zero only.

Reply:
By design.